### ISSUE TITLE
Use default heuristics if required entries are not set in rule file

### ISSUE THREAD

**Issue by manuelakuhn** [psychoinformatics-de/datalad-hirni#194]
When entries are not defined in the rule file, would it make sense to apply the default heuristic? This would work as a fall back if required entries are not defined but also make live easier for users.

An example of why this might be helpful can be found in #192

**Comment by bpoldrack** [comment]
Agreed (in a way). What I outlined here: https://github.com/psychoinformatics-de/datalad-hirni/issues/191#issuecomment-814720155
would solve this, since that is how subsequent application of several rules work.

For issue organization I'll leave this issue open, but close the other one.
